src/transformers/convert_graph_to_onnx.py

```diff
@@ -224,7 +229,11 @@ def build_shape_dict(name: str, tensor, is_input: bool, seq_len: int):
 
 
 def load_graph_from_args(
-    pipeline_name: str, framework: str, model: str, tokenizer: Optional[str] = None, **models_kwargs
+    pipeline_name: str,
+    framework: str,
+    model: str,
+    tokenizer: Optional[str] = None,
+    **models_kwargs,
 ) -> Pipeline:
```

Comment:
Honestly, that was just the `black` formatter at work. It would be better to ignore the following commits from the git blame:
- [f892e07](https://github.com/huggingface/transformers/pull/26805/commits/f892e07d99ab1f973eb850a5fb0dfc48d4f594cb)
- [b576ce2](https://github.com/huggingface/transformers/pull/26805/commits/b576ce2efb79778b2156fd972cf549dc94e176bb)